test/rule-glossary.test.js

```diff
@@ -0,0 +1,20 @@
+const globby = require('globby')
+const glossaryUsages = require('./../_data/glossary-usages.json')
```

Comment:
The tests on Circle are failing on that line, not finding the file. Is this supposed to be a new file that you forgot to commit?

Reply:
@Jym77 - darn this PR cannot be merged until the implementation PR gets merged.

The `glossary-usages.json` is dynamically generated on site build & in this branch it gets generated as a part of gatsby hook, but in the PR https://github.com/act-rules/act-rules.github.io/pull/604 this has been moved to a npm script, that runs outside the scope of gatsby build.

I do not wish to commit this file.

Ignore the CI failure. 

I have listed all the missing glossary definitions above. If we work on fixing up the actual missing definitions, that will help.

Reply:
I agree that auto-generated files should not be committed.

But merging a PR that breaks the CI tests kinda destroy the main purpose of CI tests. Especially, if a further PR is breaking the tests for real, we won’t see it immediately since we'll be in the habit that "oh yeah, tests are broken, as usual"….
So in my opinion, either this PR should use the old way of finding the file (and #604 should adapt this script once its merged), or this PR should wait until #604 is merged before being merged.

Also, this PR is going to break tests (the whole point of it being to add a test that fails on missing glossary items to find these missing items). I’m not in favour of merging without either fixing the test (filling the glossary) or ignoring them (commenting them out, or equivalent, until fix is done). We should not get used to see red lights in the checks and consider that as normal…

Reply:
Clarification @Jym77. I never said we are going to merge this PR with failing tests.

I just don't want to repeat the work done on PR here - https://github.com/act-rules/act-rules.github.io/pull/604/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R155, which is why it is best if we merge this PR after merging the other one and resolving conflicts on this.

We never did "oh yeah, tests are broken, as usual", on this project, so please don't assume that.

The intent of this PR is to find the missing glossary terms and fix them & not go around in circles for currently failing test, where we have a roadmap to fix them. Thanks.

Reply:
As a temporary measure for now have added the generated `json` file. See - https://github.com/act-rules/act-rules.github.io/pull/634/files#diff-a084b794bc0759e7a6b77810e01874f2R75 and https://github.com/act-rules/act-rules.github.io/pull/634/files#diff-f5cfbc832d01fe2cf6d74079a9346113

Reply:
Cleaned up the temporary addition, given #604 is merged.

Reply:
Thanks. Sorry for the fuss around it.